app/models/relationship.rb

```diff
@@ -0,0 +1,29 @@
+class Relationship < Term 
+
+  attr_accessor :commit_history
+
+  configure :type => RDF::URI("http://vivoweb.org/ontology/core#Relationship")
+
+  property :parents, :predicate => RDF::Vocab::SKOS.broader
```

Comment:
I think make this singular. There can be multiple of this property, but each one will only have one value.